### ISSUE TITLE
[Bug] Duplicate key error in Users table during load tests

### ISSUE THREAD

**Issue by navalev** [MicrosoftDX/nether#423]
Scenario: load test the leaderboard with 600 users
Error:
```
System.Data.SqlClient.SqlException: Violation of PRIMARY KEY constraint 'PK_Users'. Cannot insert duplicate key in object 'dbo.Users'. The duplicate key value is (loadUser9646).
.
.
.
in C:\Projects\nether\src\Nether.Data.EntityFramework\Identity\EntityFrameworkUserStore.cs:line 73
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Nether.Web.Features.Identity.UserController.<PutUser>d__5.MoveNext() in C:\Projects\nether\src\Nether.Web\Features\Identity\Controllers\UserController.cs:line 92
```

Happens on clean env with no existing record.
Note - still need to verify if originated from load test flow

**Comment by stuartleeks** [comment]
The load test is generating the usernames by concatenating a fixed prefix with a [random number](https://github.com/MicrosoftDX/nether/blob/master/NetherLoadTest/NetherLoadTest/NetherUnitTest.cs#L93) (`random.Next(10000)` will generate a random number from 0 to 10000). By my quick calculation, there's a ~1% chance of getting 300 unique usernames by this method.

It may make sense to look at a different strategy for generating the user names.

**Comment by ankodu** [comment]
@navalev why is there a **Review and Merge** label on this BUG?

**Comment by navalev** [comment]
fixed in PR #424 once merged - drop use of random for the loadtest user name. instead use the load test generated user id.
